lesson5.html

```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Lesson5 - JS</title>
+    <style>
+    
+        table, .basket, .catalog {
+            margin: 15px;
+            border-collapse: collapse;
+        }
+
+        td {
+            width: 50px;
+            height: 50px;
+            text-align: center;
+            border: 1px solid black;
+            box-sizing: border-box;
+        }
+
+        .black {
+            background-color:burlywood;
+        }
+
+        .without_border {
+            border-left: none;
+            border-bottom: none;
+        }
+
+        .title {
+            font-weight: bold;
+        }
+
+        .catalog {
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: space-between;
+            width: 650px;
+        }
+
+        .card {
+            height: 250px;
+            width: 150px;
+            border: 1px solid black;
+            text-align: center;
+            margin: 5px;
+            line-height: 10px;
+        }
+
+        .card__img {
+            width: 150px;
+        }
+        .card__link {
+            display: block;
+            text-decoration: none;
+            background-color:chartreuse;
+            width: 100px;
+            margin-left: 25px;
+            height: 25px;
+            line-height: 25px;
+        }
+        .catalog__quantity {
+            width: 25px;
+            padding-left: 8px;
+        }
+        
+    </style>
+</head>
+<body>
+    <h3>Задание №1. Шахматная доска.</h3>
+    <div class="board" id="board"></div>
+    <hr>
+    <h3>Задание №2. Корзина.</h3>
+    <div class="basket" id="basket"></div>
+    <div class="basket" id="basket-null"></div>
+    <hr>
+    <h3>Задание №3. Каталог товаров.</h3>
+    <div class="catalog" id="catalog"></div>
+
+
+
+
+
+
+    <script>
+// CHESS BOARD. 1
+
+
+        var $block = document.getElementById('board');
+        var $table = document.createElement('table');
+
+        abc = [' ', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'];
+        num = [8, 7, 6, 5, 4, 3, 2, 1]
+        
+        for(var i = 0; i < 9; i++) {
+            var $row = document.createElement('tr');
+            for(var j = 0; j < 9; j++) {
+                var $cell = document.createElement('td');
+                $row.appendChild($cell);
+                //добавляем буквы
+                if(i == 8){
+                    $cell.textContent = abc[j];
+                }
+                //добавляем цифры
+                if(j == 0){
```

Comment:
лучше использовать строгое сравнение
```suggestion
                if(j === 0){
```